mongodb-sharding-centos/azuredeploy.json

```diff
@@ -0,0 +1,884 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Administrator user name used when provisioning virtual machines"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Administrator password used when provisioning virtual machines"
+      }
+    },
+    "mongoAdminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "MongoDB Administrator user name used when provisioning MongoDB sharding cluster"
+      }
+    },
+    "mongoAdminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "MongoDB Administrator password used when provisioning MongoDB sharding cluster"
+      }
+    },
+    "dnsNamePrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "DNS Name for the publicly accessible router nodes. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "CentOSVersion": {
+      "type": "string",
+      "defaultValue": "7.0",
+      "allowedValues": [
+        "6.5",
+        "6.6",
+        "6.7",
+        "7.0",
+        "7.1",
+        "7.2"		
+      ],
+      "metadata": {
+        "description": "The CentOS version for the VM. This will pick a fully patched image of this given CentOS version."
+      }
+    },
+    "numDataDisks": {
+      "type": "string",
+      "defaultValue": "4",
+      "allowedValues": [
+        "2",
+        "3",
+        "4",
+        "5",
+        "6",
+        "7",
+        "8",
+        "9",
+        "10",
+        "11",
+        "12",
+        "13",
+        "14",
+        "15",
+        "16",
+        "32",
+        "64"
```

Comment:
We found a bug with the disk selector. We're going to remove the 64 disk options since more than 40 fully utilized disks in a storage account will cause a bottleneck.